hail/src/main/scala/is/hail/asm4s/Code.scala

```diff
@@ -824,6 +824,12 @@ class CodeLong(val lhs: Code[Long]) extends AnyVal {
 
   def compare(rhs: Code[Long]): Code[Int] = Code(lhs, rhs, lir.insn2(LCMP))
 
+  def max(rhs: Code[Long]): Code[Long] =
```

Comment:
These changes snuck in an earlier approach to fix this is in Scala before I settled on clamping in python. I can remove if you'd like

Reply:
Let's remove this since not used, unneeded